web/src/routes/OrganizationsList/Organization/Tabs/DefaultPermissions/DefaultPermissionsToolbar.tsx

```diff
@@ -52,6 +56,18 @@ export default function DefaultPermissionsToolbar(
           >
             Create default permission
           </Button>
+          <Conditional if={props.selectedItems?.length !== 0}>
+            <ToolbarItem>
+              <Button
+                style={{paddingBottom: '0px'}}
+                icon={<TrashIcon style={{fontSize: '24px'}} />}
```

Comment:
Typically it's better to use `em` or some other scalable unit so it will change size with the screen size.

Reply:
replaced it with: 
```
icon={
  <Icon size="lg">
     <TrashIcon />
  </Icon>
}
```